pkg/admission/handlers/sts_test.go

```diff
@@ -0,0 +1,103 @@
+package handlers
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/kanopy-platform/hedgetrimmer/internal/admission"
+	"github.com/kanopy-platform/hedgetrimmer/internal/limitrange"
+	"github.com/stretchr/testify/assert"
+	admissionv1 "k8s.io/api/admission/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	kadmission "sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+type MockMutator struct {
```

Comment:
We probably want this defined outside of this sts_test.go to be shared across all handler tests so we don't redefine in each, maybe in a `mocks.go` file?

If we do that, just need rename it `mockMutator`